Makefile

```diff
@@ -432,34 +420,17 @@ clean-onprem:
 ############
 
 # Current Operator version
-OPERATOR_VERSION ?= 0.0.1
-BUNDLE_OUTPUT_DIR := $(or ${BUNDLE_OUTPUT_DIR},$(BUILD_FOLDER)/bundle)
+OPERATOR_VERSION ?= 0.0.3
+BUNDLE_OUTPUT_DIR ?= deploy/olm-catalog/$(OPERATOR_VERSION)
+BUNDLE_METADATA_OPTS ?= --channels=alpha,ocm-2.3 --default-channel=alpha
 
-# Generate bundle manifests and metadata, then validate generated files.
 .PHONY: operator-bundle
-operator-bundle: create-ocp-manifests
```

Comment:
The create-ocp-manifests target can be removed because the yamls in config/assisted-service have been removed.

The --apply-manifest flag can also be removed from the deploy- targets and the use of that flag can be removed from the python scripts in ./tools.

Reply:
Done.

Reply:
Ran into a bunch of failures when I removed it and I suspect I did something wrong. Reverting that and tackling it in a follow-up PR.